### PR TITLE
Added new html and css for the building dashboard page

### PR DESCRIPTION
**Pull request by askiefer**
Hey both, wanted to go ahead and push the new html / css for the latest mockups, https://projects.invisionapp.com/share/2SAI4AK48#/screens/221959111.

Although I'm srcing in the javascript chart files the charts are returning js console errors (ts tsk on my part I know). 

This does not include the building data at all (I hardcoded one in) but I wanted to provide the html for the layout at the very least. This will also require additional css but it's a start.

### CODE REVIEW

**Comment by eayoungs** [comment]
Looks good @askiefer ! I don't see the 'donut', should I assume that's coming?

Also, a minor issue, but I don't see a link to `building_dashboard.html` in the main `index.html` file. This is the landing page that opens when you run _SimpleHTTPServer_ (which is *http.server* in Python3, BTW).
LMK whether or not you want to change either of these things and I'll be happy to merge this PR.

**Review by eayoungs — APPROVED** [review]
- Didn't see the _donut_ chart.
- [x] Might want to add a link to `building_dashboard.html` in the root `index.html` file for easy access

File: dashboard/building_dashboard.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+	                				2016 BENCHMARKS ARE DUE<br>
+	                				APRIL 1, 2017
+	                			</div>
+	                		</div>
+	                		<div class="row center">
+	                			<button type="button" class="btn btn-primary btn-round-lg btn-lg">SUBMIT NOW</button>
+	                		</div>
+	                		<div class="links">
+		                		<div class="row link">
+		                			<a href="/https://sfenvironment.org/energy/energy-efficiency/commercial-and-multifamily-properties/existing-commercial-buildings-energy-performance-ordinance">What is the Existing Commercial Buildings Ordinance?</a>
```
Comment (thfield) [inline dashboard/building_dashboard.html]: `<a href="https://sfenvironment.org/energy/ ...`

File: dashboard/building_dashboard.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+		                		</div>
+		                		<div class="row link">
+		                			<a href="/https://sfenvironment.org/sites/default/files/fliers/files/sfe_gb_ecb_benchmark_instructions.pdf">How to submit benchmarking data</a>
```
Comment (thfield) [inline dashboard/building_dashboard.html]: `<a href="https://sfenvironment.org/sites ...`

File: dashboard/building_dashboard.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+			        			<div id="chart-histogram" class="histogram-chart"></div>
+			        		</div>
+			        	</div>
+		            	<div class="panel-body bottom">
+		            		<div class="panel-bottom-header">ABOUT THE ENERGY STAR SCORE</div><br>
+		            		<div class="panel-bottom-text">
+		            		The ENERGY STAR score is a rating between 1-100 that summarizes at a glance how a building is performing compared to similar buildings nationwide, after accounting for local climate. The higher the score, the better; buildings with a score of 75 or better may qualify for ENERGY STAR certification as top performers. The score uses the information you entered about your building, including: size, location, occupancy, number of computers, etc.
+		            		</div><br>
+		            		<div class="link">
+		            			<a href="/https://sfenvironment.org/article/commercial/sf-energy-watch-commercial-program">San Francisco Energy Watch can help you save money, save energy, and improve your ENERGY STAR score</a>
```
Comment (thfield) [inline dashboard/building_dashboard.html]: `<a href="https://sfenvironment.org/article/commercial/sf-en`

File: dashboard/building_dashboard.html [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+	                			</div>
+	                		</div>
+	                		<div class="row center">
+	                			<div class="side-panel-text">
+	                				2016 BENCHMARKS ARE DUE<br>
+	                				APRIL 1, 2017
+	                			</div>
+	                		</div>
+	                		<div class="row center">
+	                			<button type="button" class="btn btn-primary btn-round-lg btn-lg">SUBMIT NOW</button>
```
Comment (thfield) [inline dashboard/building_dashboard.html]: should the button be a `<a href="http://sfenvironment.org/submit" class="btn btn-primary ...` instead, so clicking goes somewhere?

**Review by thfield — CHANGES_REQUESTED** [review]
looks great!
let's get together a list of the all dynamic elements and give them ids so we can get this using real data!

**Comment by thfield** [comment]
ps. this past week i wired up a c3 ringchart in my branch that's a part of #18, so i think we're in pretty good shape.  just a matter of connecting the pieces now

File: dashboard/building_dashboard.html (hunk 4 above)
Comment (thfield) [reply dashboard/building_dashboard.html]: this one isn't a "required" change, just something to think about

**Comment by askiefer** [comment]
@eayoungs Yes totally, will do! And yes the donut chart is to come. I'll see if I can add it.

And yes thanks @tyler - totally! Will add.

**Comment by askiefer** [comment]
Also I might incorporate the latest mockup changes in this PR - shouldn't be a lot but just an FYI.
And yes @thfield for sure - I kept the ids for the charts the same:
- ``#chart-histogram`` (for the ENERGY STAR score)
- ``#chart-stackedbar``
- ``#chart-histogram-ghg`` (histogram for the GHG score)

**Review by thfield — APPROVED** [review]

File: dashboard/building_dashboard.html [hunk 5]
```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+	<title>SF Building Dashboard</title>
+	<link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.css" />
+	<!-- Bootstrap core CSS -->
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	<script src="https://cdn.datatables.net/1.10.12/js/jquery.dataTables.min.js"></script>
```
Comment (eayoungs) [inline dashboard/building_dashboard.html]: I was getting some errors in the console about jquery not found but fixed them by adding this line here:
`<script src="http://code.jquery.com/jquery-1.11.3.min.js"></script>`

**Review by eayoungs — CHANGES_REQUESTED** [review]
I'm not seeing any of the individual charts that I thought were added. I noticed some errors in the console and found the change below eliminated several related to jQuery.
There are a few more errors related to attributes in the page having NaN values. I think it might be best to close this PR and tackle this with some further work, especially since @thfield is relying on these changes in the next PR.

File: dashboard/building_dashboard.html (hunk 5 above)
Comment (thfield) [reply dashboard/building_dashboard.html]: there are a couple repeated js libraries and a couple that are for maps which aren't being used.  i pared them down at some point in #27

**Comment by thfield** [comment]
i don't think the charts actually get implemented in this PR, i'm about to tackle the GHG histogram, and i think that @askiefer got the barchart working locally?

**Review by eayoungs — APPROVED** [review]

**Comment by eayoungs** [comment]
Looks good. I see charts on _ENERGY STAR SCORE_ and _SITE ENERGY USE INTENSITY_.

**Pull request merged**